src/main/java/com/tagtraum/perf/gcviewer/ctrl/action/Export.java

```diff
@@ -96,31 +96,31 @@ public void exportFile(final GCModel model, File file, final String extension, f
         }
     }
 
-    private static class ExtensionFileFilter extends FileFilter {
-        private String extension;
-        private String description;
-        private DataWriterType dataWriterType;
+    private static class ExportExtensionFileFilter extends ExtensionFileFilter {
+
+        public static final ExportExtensionFileFilter[] EXT_FILE_FILTERS = {
+            new ExportExtensionFileFilter(".csv", LocalisationHelper.getString("fileexport_dialog_csv"), DataWriterType.CSV),
```

Comment:
Thank you very much - I have merged your pull request and added you to 
the list of contributors.